### PR TITLE
Update dependency-injection.md

### LINKED ISSUE #22559: Confusing language describing the different lifespans
There seems to be an error in language regarding the lifespans, just above the section ["Call services from main"](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/dependency-injection?view=aspnetcore-5.0#call-services-from-main)...

The document states: "Scoped objects are the same for each request but different across each request."

I assume it's instead meant to be: ""Scoped objects are the same for each request but different across each **session**."? (or something similar to that nature)

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 0d78f540-81fd-1ba9-cb90-591e4ec27d9b
* Version Independent ID: 2251f557-dd3a-ec35-2558-6243337db009
* Content: [Dependency injection in ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/dependency-injection?view=aspnetcore-5.0#lifetime-and-registration-options)
* Content Source: [aspnetcore/fundamentals/dependency-injection.md](https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/fundamentals/dependency-injection.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-fundamentals**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

### PR DESCRIPTION
**Pull request by steveberdy**
## Summary

Fixes ambiguity in description of the lifetime of scoped objects in dependency injection.

Fixes #22559

### CODE REVIEW

**Comment by guardrex** [comment]
Hi @steveberdy ... I don't recommend the use of the word "session" in this context. It's overloaded given the meaning and use of "[session state](https://docs.microsoft.com/aspnet/core/fundamentals/app-state)". I recommend that if any changes are made that they avoid the use of the word "session."

**Comment by steveberdy** [comment]
I made that change, but I'm not sure if that makes it any less confusing for documentation readers

**Comment by guardrex** [comment]
I don't think it's confusing in either form, but maybe I'm wrong about that. What's confusing about it? On a single request, the object is the same. Across different requests, there are different (new) objects. We've had this language up for *many years* with very few (none until now?) issues opened on it.

**Comment by steveberdy** [comment]
There should be a way to say that scoped objects have the same content for each request, but are different objects for different requests. It's not really that confusing, but it has potential to be confusing. It is fine as is now, but could use more for an abundance of clarity.

**Comment by steveberdy** [comment]
@Rick-Anderson could you provide us with any feedback on this edit before merging?

**Review by Rick-Anderson — APPROVED** [review]

File: aspnetcore/fundamentals/dependency-injection.md [hunk 1]
```diff
@@ -220,7 +220,7 @@ Scoped services must be resolved in the `InvokeAsync` method:
 The logger output shows:
 
 * *Transient* objects are always different. The transient `OperationId` value is different in the `IndexModel` and in the middleware.
-* *Scoped* objects are the same for each request but different across each request.
+* *Scoped* objects are the same for each request but different across requests.
```
Comment (Rick-Anderson) [inline aspnetcore/fundamentals/dependency-injection.md:223]:
```suggestion
* *Scoped* objects are the same for a given request but differ across each new request.
```

**Pull request merged**